NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/QuestionnaireTestOptionActivity.cs

```diff
@@ -13,7 +13,7 @@
 using NDB.Covid19.Droid.Services;
 using NDB.Covid19.Droid.Utils;
 using NDB.Covid19.Enums;
-using NDB.Covid19.Utils;
+using NDB.Covid19.Utils; 
```

Comment:
remove extra whitespace

Reply:
![image](https://user-images.githubusercontent.com/14860255/153426115-383f7074-dd59-4de0-9d78-579b18a72b11.png)

Reply:
Just fix it.
Also, remove these unused import statements in this file since you will be cleaning up in it anyway:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/14860255/153550562-e7877edd-0bfd-48ae-8fca-ca4554e35fef.png">